### PR TITLE
VTCCA-4292-terms-con-page

### PR DESCRIPTION
**Pull request by zahiriqbal**

### CODE REVIEW

File: test/controllers/StaticPagesControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import models.DetailedIndividualAccount
+import org.scalacheck.Arbitrary._
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import utils.{StubIndividualAccountConnector, _}
+import views.html.createAccount.termsAndConditions
+
+class StaticPagesControllerSpec extends VoaPropertyLinkingSpec {
+
+  private object TestRegistrationController$
```
Comment (PeterPerhac) [inline test/controllers/StaticPagesControllerSpec.scala:28]: why is there a dollar `$` sign in the name?

File: test/controllers/StaticPagesControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      extends StaticPagesController(stubMessagesControllerComponents(), termsAndConditionsView)
+
+  "Static page" should
+    "find terms and conditions page and returns 200" in {
+    val (groupId, externalId): (String, String) = (shortString, shortString)
```
Comment (PeterPerhac) [inline test/controllers/StaticPagesControllerSpec.scala:33]: what does this line do?

File: test/controllers/StaticPagesControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import models.DetailedIndividualAccount
+import org.scalacheck.Arbitrary._
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import utils.{StubIndividualAccountConnector, _}
+import views.html.createAccount.termsAndConditions
+
+class StaticPagesControllerSpec extends VoaPropertyLinkingSpec {
+
+  private object TestRegistrationController
```
Comment (darrell-plant) [inline test/controllers/StaticPagesControllerSpec.scala:28]: this is not a `RegistrationController` (copy and paste mistake)

**Comment by darrell-plant** [comment]
LGTDz 👍 (after couple of small comments in the test)

**Pull request merged**